### PR TITLE
Feature/ahti 310 new

### PR DESCRIPTION
**Pull request by marharyta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "@types/mapbox-gl": "^1.8.0",
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
+    "@types/react-burger-menu": "^2.6.1",
```
Comment (ahie) [inline package.json]: These are unused? Remove?

File: src/common/ui-components/Menu/Menu.tsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ const translateMenuCategories = (
 };
 
 const Menu: React.FC<MenuProps> = ({
-  className,
```
Comment (ahie) [inline src/common/ui-components/Menu/Menu.tsx]: Why is className removed from props?

File: src/common/ui-components/MenuIcon/MenuIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <div
+      className={classNames(
+        isOpen
+          ? styles.navIcon3Open
+          : isDark
+          ? styles.navIcon3Dark
+          : styles.navIcon3
+      )}
+    >
+      <span></span>
```
Comment (ahie) [inline src/common/ui-components/MenuIcon/MenuIcon.tsx]: Not the biggest fan of this, seems hacky. But the end result looks good.

File: src/common/ui-components/WrappedMenu/WrappedMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (ahie) [inline src/common/ui-components/WrappedMenu/WrappedMenu.tsx]: Looks like this file should be in domain and not common as it touches the apps state.

Comment (ahie) [reply src/common/ui-components/WrappedMenu/WrappedMenu.tsx]: A better name would be AhtiMenu?

File: src/domain/Ahti/Ahti.module.scss [hunk 5]
```diff
@@ -9,14 +9,14 @@ $menu-z-index: 1000;
 }
 
 .menu {
-  position: sticky;
+  // position: sticky;
```
Comment (ahie) [inline src/domain/Ahti/Ahti.module.scss]: Remove.

File: src/domain/Ahti/Ahti.module.scss [hunk 6]
```diff
@@ -9,14 +9,14 @@ $menu-z-index: 1000;
...
   top: 0;
   background-color: var(--hds-ui-color-white);
   z-index: $menu-z-index;
 }
 
 .search {
-  width: $search-component-width;
+  // width: $search-component-width;
```
Comment (ahie) [inline src/domain/Ahti/Ahti.module.scss]: Remove.

File: src/domain/ContentPage/ContentPage.tsx [hunk 7]
```diff
@@ -3,19 +3,25 @@ import { IconCheck, IconClose } from 'hds-react';
 import { useTranslation } from 'react-i18next';
 import classNames from 'classnames/bind';
 
+// import Menu, { MenuItem } from '../../common/ui-components/Menu/Menu';
```
Comment (ahie) [inline src/domain/ContentPage/ContentPage.tsx]: Please remove.

File: package.json (hunk 1 above)
Comment (marharyta) [reply package.json]: true

File: src/common/ui-components/Menu/Menu.tsx (hunk 2 above)
Comment (marharyta) [reply src/common/ui-components/Menu/Menu.tsx]: hmm, it was not used?

File: src/common/ui-components/MenuIcon/MenuIcon.tsx (hunk 3 above)
Comment (marharyta) [reply src/common/ui-components/MenuIcon/MenuIcon.tsx]: of what exactly?

File: src/common/ui-components/WrappedMenu/WrappedMenu.tsx (hunk 4 above)
Comment (marharyta) [reply src/common/ui-components/WrappedMenu/WrappedMenu.tsx]: yes, I agree. How abour I do it in the next PR?

File: src/domain/Ahti/Ahti.module.scss (hunk 5 above)
Comment (marharyta) [reply src/domain/Ahti/Ahti.module.scss]: agree

File: src/domain/Ahti/Ahti.module.scss (hunk 6 above)
Comment (marharyta) [reply src/domain/Ahti/Ahti.module.scss]: yep.

File: src/domain/ContentPage/ContentPage.tsx (hunk 7 above)
Comment (marharyta) [reply src/domain/ContentPage/ContentPage.tsx]: yep

**Pull request merged**